Presentr/Interactr.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  Interactr.swift
+//  Presentr
+//
+//  Created by Andreas Grauel on 26.07.17.
+//  Copyright © 2017 danielozano. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+public class Interactr: UIPercentDrivenInteractiveTransition {
+    public enum Direction {
+        case up
+        case down
+        case left
+        case right
+    }
+    
+    var hasStarted = false
+    var shouldFinish = false
+    
+    public var percentThreshold: CGFloat = 0.3
+    
+    public func calculateProgress(_ translationInView:CGPoint, viewBounds:CGRect, direction:Direction) -> CGFloat {
```

Comment:
@mrgrauel try to be consistent. On the line 23 you added one style for the type (adding a space), now you didn't do it. Keep the same standard all over your code.